src/modules/featureInfo/components/featureInfoPanel.css

```diff
@@ -112,3 +113,7 @@ img {
 .ba-section:hover .ba-list-item__header .ba-list-item__text {
 	color: var(--primary-color) .3;
 }
+
+.grundsteuer p {
+	padding: 0 0 .8em 0;
+}
```

Comment:
Let's add a comment, something like "just for grundsteuer release, should be removed later"

Reply:
done